src/test/api_compatibility.ts

```diff
@@ -5,6 +5,7 @@ import { BinaryCarrier, FORMAT_BINARY, FORMAT_TEXT_MAP, Reference, REFERENCE_CHI
 export interface ApiCompatibilityChecksOptions {
     /** a boolean that controls whether or not to verify baggage values */
     checkBaggageValues?: boolean;
+    checkInjectBehaviour?: boolean;
```

Comment:
by default these checks should be on, so a better name for the option is `skipInjectExtractTests`